tests/unit/utils/test_logger.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import os
+
+from utils.logger import Logger
+
+log_file_name = "test.log"
```

Comment:
use a path in TMPDIR, e.g.: https://docs.python.org/3/library/tempfile.html

Reply:
(the motivation being we don't want to dump random files into the users working directory, even if we successfully clean up it could be stomping an existing file, and if we don't clean up it's littering the working directory)

Reply:
ah yes, that is good idea, indeed.

Reply:
updated

Reply:
@bparees ^